src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,17 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
-
+# Documentation for `report-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 report-mapping-annotations:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
 update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+config/%_term_exclusions.txt: config/%_term_exclusions.tsv mirror/%.owl
```

Comment:
I'm assuming this is the correct place in the makefile to add this make goal.

### Issues
Make goal seems to fail because of absence of these goals:
1. `config/%_term_exclusions.tsv`
2. `mirror/%.owl`

Reply:
YES! Awesome work I have not looked at everything yet but super excited so far!

Reply:
figured this one out. going to try on my own and if theres an issue ill report back